locales/en-US/webextension.properties

```diff
@@ -53,3 +53,5 @@ termsAndPrivacyNotice2 = By using Firefox Screenshots, you agree to our {termsAn
 termsAndPrivacyNoticeTermsLink = Terms
 # This string is used as the text for a link in termsAndPrivacyNoticeCloudServices
 termsAndPrivacyNoticyPrivacyLink = Privacy Notice
+# This string is used to label the item in the Library panel that opens the screenshots page
+libraryLabel = Screenshots
```

Comment:
Again, I just went with what the mock said to do. Happy to make changes. Aaron Benson probably has more context here. However, I will say this:

- All the other items in this menu ("Bookmarks", "History", "Downloads", "Synced Tabs", "Pocket List") have no possessive. I think it might be odd if this one stood out, and given we are consistent about not using possessives elsewhere in Firefox menus, I don't think we should change that here.
- Just 'Shots' with no context I think would be less clear (ie as a user, I'd wonder what that was...) than "Screenshots"